### PR TITLE
Fixed Addres book header not showing

### LINKED ISSUE #4611: Title 'Address book' is missing and button 'Add New Address' is shifted up in Address book if user opens it via Manage Addresses link
**Description**:
Title 'Address book' is missing and button 'Add New Address' is shifted up in Address book in My Account when opening it via Manage Addresses link

**Expected behavior**:
Title is displayed as in screenshot
Screenshot 2022-03-29 at 15 49 57

**To Reproduce**:
1. Log in
2. Open My Account
3. Click on Manage Addresses

**Actual result**:
Title 'Address book' is missing and button 'Add New Address' is shifted up 
Screenshot 2022-03-29 at 15 49 28

### PR DESCRIPTION
**Pull request by SalimMersally**
**Related issue(s):**
* Fixes #4611 

**Problem:**
* Title 'Address book' is missing and button 'Add New Address' is shifted up in Address book in My Account when opening it via Manage Addresses link

**In this PR:**
* Fixed header not showing

### CODE REVIEW

**Comment by carinadues** [comment]
Seems good. This is how it looks for me:
image

**Comment by AleksandrsKondratjevs** [comment]
Breaks orders. 
image

**Comment by aleksandrakorolova** [comment]
Still, can reproduce issue if user doesn't have default shipping/billing address and clicks on Go to "Address Book", to configure them!
Screenshot 2022-04-04 at 12 12 19

File: packages/scandipwa/src/component/MyAccountAddressBook/MyAccountAddressBook.container.js [hunk 1]
```diff
@@ -40,6 +40,15 @@ export class MyAccountAddressBookContainer extends PureComponent {
         showCreateNewPopup: this.showCreateNewPopup.bind(this)
     };
 
+    __construct(props) {
+        super.__construct(props);
+
+        const { changeTabName, tabMap } = props;
+        const { tabName } = tabMap[ADDRESS_BOOK];
+
+        changeTabName(tabName);
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/MyAccountAddressBook/MyAccountAddressBook.container.js:49]: Please don't make such actions inside the constructor.
The purpose of the constructor in React components is to initialize values.
Please perform all changes inside the lifecycle methods.

File: packages/scandipwa/src/component/MyAccountDashboard/MyAccountDashboard.container.js [hunk 2]
```diff
@@ -36,7 +36,6 @@ export class MyAccountDashboardContainer extends PureComponent {
 
     containerProps() {
         const { customer } = this.props;
-
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/MyAccountDashboard/MyAccountDashboard.container.js:39]: Please put the empty line between the `return` statement and the rest of the code.

**Review by zans-laksa — CHANGES_REQUESTED** [review]

**Review by zans-laksa — APPROVED** [review]

**Pull request merged**